code_challenges/code_wars/increment_string.py

```diff
@@ -0,0 +1,31 @@
+def increment_string(characters):
+    if characters == "":
+        return "1"
+    if characters[-1].isalpha():
+        characters += "1"
+    else:
```

Comment:
Instead of nesting all the relevant code in an else block, how about adding a return statement to this guard clause.